ouroboros-network/src/Ouroboros/Network/Protocol/Codec/Cbor.hs

```diff
@@ -59,7 +64,39 @@ cborDecoder decoder = Fold (idecode [] =<< CBOR.deserialiseIncremental decoder)
                   -- the end of stream is given, and so the user is in limbo with
                   -- neither a success, nor a failure...
                   CBOR.Partial _ ->
-                    pure $ Left $ pack "CBOR decoder is still partial after end of input stream"
+                    pure $ Left $ packError "CBOR decoder is still partial after end of input stream"
               , more = \bss -> Fold $ idecode bss term
               }
       in  feedInputs inputs k
+
+convertCborEncoder :: (a -> CBOR.Encoding) -> a -> [ByteString]
+convertCborEncoder cborEncode =
+    LBS.toChunks
+  . toLazyByteString
+  . CBOR.toBuilder
+  . cborEncode
+
+{-# NOINLINE toLazyByteString #-}
+toLazyByteString :: BS.Builder -> LBS.ByteString
+toLazyByteString = BS.toLazyByteStringWith strategy LBS.empty
+  where
+    strategy = BS.untrimmedStrategy 800 LBS.smallChunkSize
```

Comment:
Why `800`? I have no idea whether that's good or bad, just wondering. Shouldn't it be a power of 2? @dcoutts

Reply:
I will merge it as is, and we'll have a chance to ask @dcoutts why `800` is so special at today's meeting.